src/codeu/chat/server/contexts/MessageContext.java

```diff
@@ -0,0 +1,22 @@
+package codeu.chat.server.contexts;
+
+import codeu.chat.common.BasicView;
+import codeu.chat.common.Message;
+
+public class MessageContext extends codeu.chat.contexts.MessageContext {
```

Comment:
why invent this and Context classes if they are identical to the MessageContext and Context they inherit from?